src/sentry/search/events/fields.py

```diff
@@ -441,6 +442,7 @@ def normalize_count_if_value(args: Mapping[str, str]) -> Union[float, str]:
             ),
             result_type="boolean",
         ),
+        PseudoField(TRANSACTION_STATUS_ALIAS, TRANSACTION_STATUS_ALIAS),
```

Comment:
I'm not sure we need this, Transaction_status isn't a pseudo field (ie. a value we treat as a field, but doesn't actually exist in clickhouse)

Reply:
Ahh you're right. removed!